docs/network protocol.md

```diff
@@ -366,15 +367,22 @@ Used to request a single or multiple values from the server's data storage, see
 | keys | list\[str\] | Keys to retrieve the values for. |
 
 Additional arguments sent in this package will also be added to the [Retrieved](#Retrieved) package it triggers.
+Some special keys exist with specific return data:
+
+| Name                       | Type                  | Notes                                        |
+|----------------------------|-----------------------|----------------------------------------------|
+| \_read_hints_{team}_{slot} | list\[[Hint](#Hint)\] | All Hints belonging to the requested Player. |
```

Comment:
is it a good idea to be able to read other teams' hints?

Reply:
I don't really know. Currently teams don't even work, so it's hard to say.